### DISCUSSION TITLE
How to set language in messenger.send_button() ?

### DISCUSSION THREAD

**Discussion by terrarishabh** [filipporomani/whatsapp#51] [Q&A] [answered]
My code is like this:
```
messenger.send_button(
            recipient_id=mobile_number,
            lang="id_ID",
            button={}
)
```

But I get this error: `TypeError: send_button() got an unexpected keyword argument 'lang'`

According to [this file](https://github.com/filipporomani/whatsapp-python/blob/58de722d3b3889499ff8bf9161daef8a7d0b883a/whatsapp/ext/_message.py#L36) I can set the `lang` arg to set the language. Is this also possible with `send_button()` .

Because right now my whatsapp bot doesn't send messages in any language other than english, I think I have to set a lang field with send_button for other languages to work?

**Comment by filipporomani** [comment, marked as answer]
Hello!
Unfortunately, the lang karg is only available in the send_template method, which needs a language to be specified as the template you send must match the one approved by Meta. 
In all the other functions, where the messages are not predefined but different each time, you can change the language by writing the message in the language you want to use.
